source/about/mattermost-v10-changelog.md

```diff
@@ -4,6 +4,80 @@
 ```{include} common-esr-support-upgrade.md
 ```
 
+(release-v10.8-feature-release)=
+## Release v10.8 - [Feature Release](https://docs.mattermost.com/about/release-policy.html#release-types)
+
+**Release day: 2025-05-16**
+
+### Important Upgrade Notes
+ - New tables ``AccessControlPolicies`` and ``AcessControlPolicyHistory`` will be created. The migration is fully backwards-compatible, non-locking, and zero downtime is possible.
```

Comment:
> and zero downtime is possible.

I find this to be ambiguous. Will there be zero down-time or not?  If it's merely "possible" to have zero-down time, is there something an admin is supposed to do to make "possible" a certainty?